packages/pds/src/services/repo/blobs.ts

```diff
@@ -70,6 +80,74 @@ export class RepoBlobs {
     await Promise.all(blobPromises)
   }
 
+  async deleteDereferencedBlobs(did: string, writes: PreparedWrite[]) {
+    const deletes = writes.filter(
+      (w) => w.action === WriteOpAction.Delete,
+    ) as PreparedDelete[]
+    const updates = writes.filter(
+      (w) => w.action === WriteOpAction.Update,
+    ) as PreparedUpdate[]
+    const uris = [...deletes, ...updates].map((w) => w.uri.toString())
+    if (uris.length === 0) return
+
+    const deletedRepoBlobs = await this.db.db
+      .deleteFrom('repo_blob')
+      .where('did', '=', did)
+      .where('recordUri', 'in', uris)
```

Comment:
Based on the tests I am pretty sure this must be working, but what's the right way to think about updates that preserve a blob reference?  Naively it looks like the repo_blobs are being deleted for those updates here, pretty sure I am just missing something!

Reply:
oh wait think i misinterpreted question: this will delete the repo_blob linkage, (but not the blob itself) & then that repo_blob linkage gets set back up in the rest of `processWriteBlobs`

the commit value needs to be updated anyway, so we just delete & re-insert